docs/modules/operation/pages/events/event-debugging.adoc

```diff
@@ -25,18 +42,17 @@ For example:
 * open a ticket in your integrated ticketing system
 * order new ink cartridges when your printer is out of toner
```

Comment:
The sentences in this list should be capitalized and have a period at the end, but I can't offer suggestions for half of them through the GitHub diff editor.